### PR TITLE
Separate Wonday compatibility APIs CPM-3

### PR DESCRIPTION
**Pull request by terryyoon-pdftron**
https://linear.app/pdftron/issue/CPM-3/move-wonday-compatibility-apis-to-be-separate-from-main-apis

### CODE REVIEW

**Review by sgong-pdftron — COMMENTED** [review]
is that it? somehow i feel there should be a lot more, maybe some of them are not mentioned correctly

**Comment by terryyoon-pdftron** [comment]
this is what i found after searching the API.md file and past PRs for "Wonday", not sure if there is more

**Review by sgong-pdftron — CHANGES_REQUESTED** [review]
pretty sure one of the `onZoomChanged` or `onScaleChanged` is also wonday related as they are the same, which leads me thinking there are more

File: WondayAPI.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
+# Wonday Compatibility API
+
```
Comment (JamesTron) [inline WondayAPI.md:2]:
```suggestion
These APIs are convenience APIs for those migrating from a Wonday implementation.
```

**Review by JamesTron — APPROVED** [review]

**Pull request closed without merging**